src/test/java/cloud/localstack/CWMetricsTest.java

```diff
@@ -57,4 +62,45 @@ public void testCWMetricsAPIs() throws ParseException {
         ListMetricsResult metrics = cw.listMetrics(new ListMetricsRequest());
         Assert.assertEquals(metrics.getMetrics().size(), 1);
     }
+
+    @Test
+    public void testCWGetMetricData() {
+        final AmazonCloudWatch cw = AmazonCloudWatchClientBuilder.standard()
+                .withEndpointConfiguration(new AwsClientBuilder.EndpointConfiguration("http://localhost:4582", "us-east-1"))
```

Comment:
Can we use the utility methods from `TestUtils`, instead of hardcoding the URL `http://localhost:4582` here?

https://github.com/localstack/localstack-java-utils/blob/7c68e76364ab3d40f4f46650140cafea99d74cde/src/main/java/cloud/localstack/TestUtils.java#L174-L178